docs/requirements.md

```diff
@@ -0,0 +1,52 @@
+# BMC & Node Functional and Quality Requirements
+
+
+### Version
+| Date  |  Author | Content  |
+|---|---|---|
+|  2022-02-08 | Andrew Nieuwsma  | Initial Draft  |
+
+
+## Overview
+
+This document describes the functional and quality requirements that CSM expects any Redfish based BMC (baseboard management controller) controlling nodes to support.  This document is intended to give objective measurable requirements that correspond with the [profiles](../profiles/) listed.
+
+Some of these requirements transcend BMC requirements and relate to the interconnection between BMC and nodes. 
+
+## Definitions:
+
+1. Redfish - Protocol developed by [DMTF](https://www.dmtf.org/standards/redfish)
+2. BMC - Baseboard Management Controller, hereafter referred to as the 'BMC', this entity provides the Redfish interface and controls the node.
+3. Node - the hardware component that is controlled by the BMC, hereafter referred to as the 'node'.
+4. Customer - The Cray System Management (CSM) services is the consumer of the Redfish API hosted on BMCs.  The customer refers to CSM and its usage pertaining to the Redfish APIs on the BMC.
+
+
+## Functional Requirements
```

Comment:
At one time, we were tracking a customer requirement that BMCs could support x509 certificate (server, TLS) enrollment via a site-supplied CA. Is this still a requirement (recalling we're awaiting similar functionality from other MFGs)?

Reply:
Agree, good catch.

Reply:
I just added: 
``` 
| FR10 | The BMC shall support x509 certificate (server, TLS) enrollment via a customer-supplied certificate authority. | | 
```

Reply:
I would also like to add "and shall not require a Certificate Signing Request to do so".